### ISSUE TITLE
int() argument must be a string, a bytes-like object or a number, not 'NoneType' Failed to connect, try again later

### ISSUE THREAD

**Issue by aditidankar** [knowbee/llvd#46]
I am trying to download a course, I get this error:

int() argument must be a string, a bytes-like object or a number, not 'NoneType'
Failed to connect, try again later

![image](https://user-images.githubusercontent.com/20872556/117098979-92cf8680-ad8d-11eb-85ee-19617f459984.png)

**Comment by knowbee** [comment]
can you share the course link ?

**Comment by adsumit** [comment]
> can you share the course link ?

```
llvd -c "negotiating-your-job-offer" --cookies --caption
Using cookie info from cookies.txt

Current: 02-Introduction/01-Welcome.mp4 @720p
Skipping already existing video...

Current: 03-What Happens Before You Get an Offer/01-Research the position and salary range.mp4 @720p
Skipping already existing video...

Current: 03-What Happens Before You Get an Offer/02-Get clear on what you want.mp4 @720p
Skipping already existing video...

Current: 03-What Happens Before You Get an Offer/03-Establish a salary range with the employer.mp4 @720p
Skipping already existing video...

Current: 04-Receiving and Evaluating an Offer/01-What to do when you receive an offer.mp4 @720p
Skipping already existing video...

Current: 04-Receiving and Evaluating an Offer/02-Common terms and factors in an offer.mp4 @720p
Skipping already existing video...

Current: 04-Receiving and Evaluating an Offer/03-Evaluate total compensation.mp4 @720p
Skipping already existing video...

Current: 04-Receiving and Evaluating an Offer/04-What's most negotiable in a job offer.mp4 @720p
Downloading video... : 100%|████████████████████████████████████████████████████████| 109M/109M [01:30<00:00, 1.20MB/s]
Downloading subtitles..

Current: 05-How to Negotiate Salary/01-Prepare questions and counter points.mp4 @720p
int() argument must be a string, a bytes-like object or a number, not 'NoneType'
Failed to connect, try again later
```

**Comment by adsumit** [comment]
> can you share the course link ?
[](https://www.linkedin.com/learning/negotiating-your-job-offer/what-s-most-negotiable-in-a-job-offer)

**Comment by adsumit** [comment]
This course slug has the same issue: "negotiating-your-job-offer"
@knowbee

**Comment by aditidankar** [comment]
> 
> 
> can you share the course link ?

Crafting Dynamic Characters:
https://www.linkedin.com/learning/crafting-dynamic-characters/what-to-expect?u=107510546

**Comment by knowbee** [comment]
This issue is caused by python request timeout...just delete the file ```llvd``` failed to download and retry again

**Comment by knowbee** [comment]
This issue should be fixed by pr #50

**Comment by knowbee** [comment]
https://pypi.org/project/llvd/2.2.3/
